nemo/lightning/megatron_parallel.py

```diff
@@ -1827,7 +1831,7 @@ def masked_token_loss(tensor: Tensor, mask: Tensor):
     num_valid_tokens = loss_mask.sum()
```

Comment:
remove num_valid_tokens if no longer used here, and how will we handle the `num_valid_tokens=0` cases in the future? to avoid divide by 0

Reply:
both good points!